src/main/java/core/basesyntax/dao/FileReaderImpl.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax.dao;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class FileReaderImpl implements FileReader {
+
+    @Override
+    public List<String> readFile(String fileName) {
+        List<String> lines = new ArrayList<>();
+        try (BufferedReader br = new BufferedReader(new java.io.FileReader(fileName))) {
+            String line;
+            br.readLine();
```

Comment:
It seems like you are skipping the first line of the file without any checks. If the file format guarantees that the first line should always be skipped (e.g., headers), please ensure that this behavior is documented. If this is not the case, consider handling the first line appropriately.